src/components/Composer/Editor/index.tsx

```diff
@@ -13,7 +13,7 @@ import { usePublicationStore } from 'src/store/publication';
 
 import MentionsPlugin from '../../Shared/Lexical/Plugins/AtMentionsPlugin';
 
-const TRANSFORMERS = [...TEXT_FORMAT_TRANSFORMERS];
+const TRANSFORMERS = [...TEXT_FORMAT_TRANSFORMERS, LINK];
```

Comment:
@bigint going forward we have to decide if we want to convert all links to markdown link node or not `[]()`. If we do convert to link node we won't need custom transformer while preview. If we dont add this we can actually add link regex to preview transformer and convert it to link (Link ive done)

Reply:
Actually we need both, some apps in lens ecosystem will just post link some post markdown link, so we need to handle all edge cases 🙇🏼